### DISCUSSION TITLE
Copilot Chat Not Working for New Users on VS 2022

### DISCUSSION THREAD

**Discussion by PeteBaker1** [community/community#69216] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When I install the extension, it comes up saying I need to sign up for the private preview, but when I click it the link is broken as the preview has ended, so then I can't use Copilot Chat, apparently it should be public now

**Comment by LadyKerr** [comment]
@PeteBaker1 do you have a subscription to GitHub Copilot?
You need access to GitHub Copilot in order to use GitHub Copilot Chat

**Reply by PeteBaker1** [reply]
Yeah I have GitHub Copilot and that works exactly as expected :)
